server/app/controllers/fusor/api/v21/deployments_controller.rb

```diff
@@ -122,6 +122,35 @@ def validate_cdn
       end
     end
 
+    def check_mount_point
+      mount_address = params['address']
+      mount_path = params['path']
+      mount_type = params['type']
+
+      begin
+        mount_storage(mount_address, mount_path, mount_type)
+        render json: { :mounted => true }, status: 200
+      rescue => error
+        message = 'Malformed request'
+        message = error.message if error.respond_to?(:message)
```

Comment:
Ah, thank you. This was left over from before we refactored the front end. Will change.